deps/AssemblyProcessor/netstandard2.0/Stride.Core.AssemblyProcessor.deps.json

```diff
@@ -9,7 +9,7 @@
     ".NETStandard,Version=v2.0/": {
       "Stride.Core.AssemblyProcessor/1.0.0": {
         "dependencies": {
-          "Microsoft.Build.Utilities.Core": "17.3.2",
+          "Microsoft.Build.Utilities.Core": "17.0.0",
```

Comment:
should that happen? dont know why it changed this file

Reply:
This file shouldn't even be there (less in git). It is only used internally by dotnet during build and has no impact on runtime.